problems/arrayMethods.js

```diff
@@ -91,15 +149,29 @@ const targetCount = () => {};
  * @returns {Object} A mapping of elements to their occurrence count.
  */
 
-const counterObject = () => {};
+const counterObject = (items) => {
+  let obj = {}
+  items.forEach((item) => {
+    if (obj[item] = obj[item]? obj[item] + 1 : 1){
+    }  
+  })
+return obj
+};
 
 /**
  * Takes in an array of numbers and returns a new array with each element doubled.
  * @param {number[]} nums
  * @returns {number[]} Each element has been doubled
  */
 
-const doubled = () => {};
+const doubled = (nums) => {
+let output = []
+let doubled = 2
+nums.map((el) => {
+  output.push(doubled * el)
```

Comment:
map returns an array so you don't need to push into a separate array, you can just return nums.map